doc/glossary.rst

```diff
@@ -955,6 +955,20 @@ such as:
         and do not provide :term:`set_params` or :term:`get_params`.
         Parameter validation may be performed in ``__init__``.
 
+    cross-validation estimator
+        An estimator that has built-in cross-validation capabilities to
+        automatically select the best hyper-parameters, e.g.
+        :class:`ElasticNetCV <linear_model.ElasticNetCV>`, or
+        :class:`LogisticRegressionCV <linear_model.LogisticRegressionCV>`.
+        Cross-validation estimators are named *EstimatorCV*. The advantage of
```

Comment:
It might be helpful to make this more concrete, with a statement like:
```
That is, ``EstimatorCV()`` tends to be roughly equivalent to
``GridSearchCV(Estimator(), {'parm': [0, 1, 2, 3, ...]})``
```

Then again, maybe that adds confusion.